src/map/ai/controllers/mob_controller.cpp

```diff
@@ -620,8 +620,9 @@ void CMobController::Move()
         }
     }
 
-    bool  move         = PMob->PAI->PathFind->IsFollowingPath();
-    float attack_range = PMob->GetMeleeRange();
+    bool  move            = PMob->PAI->PathFind->IsFollowingPath();
+    float attack_range    = PMob->GetMeleeRange();
+    float closureDistance = attack_range - 0.2f - (static_cast<float>(PMob->getMobMod(MOBMOD_ENCROACH_TARGET)) / 10.0);
```

Comment:
I think it's less intuitive when the `-0.2` stacks with the mod, which as why I suggested:
> if zero, use the default, else take mobmod and convert to float

and I believe you want that `10.0` to be `10.0f` but the compiler might have automatic figured out your intent anyway.